src/test/scala/ai/privado/languageEngine/java/audit/DataElementDiscoveryTestBase.scala

```diff
@@ -25,7 +25,9 @@ abstract class DataElementDiscoveryTestBase extends AnyWordSpec with Matchers wi
     outputDir = File.newTemporaryDirectory()
 
     val config =
-      Config(fetchDependencies = true).withInputPath(inputDir.pathAsString).withOutputPath(outputDir.pathAsString)
+      Config(fetchDependencies = true, delombokMode = Some("no-delombok"))
```

Comment:
Do we have lombok code in test cases which extend this base class?

Reply:
No, only `DataElementDiscoveryTest` has the lombook code that for checking audit implementation only.